package.json

```diff
@@ -33,8 +34,11 @@
     "browserify": "^13.3.0",
     "eslint": "^3.14.0",
     "eslint-config-homeoffice": "^2.1.0",
+    "funkie": "0.0.5",
+    "funkie-phantom": "0.0.1",
     "nodemon": "^1.11.0",
-    "npm-run-all": "^4.0.1"
+    "npm-run-all": "^4.0.1",
```

Comment:
Unless my eyes are deceiving me, I'm using it here https://github.com/UKHomeOffice/right-to-rent-check/pull/13/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L5